core-react/src/main/VerifyAuth.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { StringParam, useQueryParam } from 'use-query-params';
+import BoxPage from './components/boxPage/BoxPage';
+import BoxPageHeader from './components/boxPage/BoxPageHeader';
+import BoxPageItems from './components/boxPage/BoxPageItems';
+import ScopeContainer from './components/scope/ScopeContainer';
+import Scope from './components/scope/Scope'
+import Form from './components/form/Form';
+import FormInputContainer from './components/form/FormInputContainer';
+import { fetchEndpoint, parseJson } from './Utils';
+
+const API_URI = process.env.NODE_ENV === 'production' ? process.env.REACT_APP_API_URI : 'http://localhost:8080/'
+const AUTH_REQUEST_ENDPOINT = `${API_URI}api/auth/request`
+
+interface PathParams {
+    authReqId: string
+}
+
+interface AuthClient {
+    client_id: string,
+    client_name: string,
+    client_url: string
+}
+
+interface AuthScope {
+    scope_id: string,
+    scope_name: string,
+    scope_description: string
+}
+
+interface AuthData {
+    auth_req_id: string,
+    expires_on: Date,
+    client: AuthClient,
+    scopes: AuthScope[],
+    verify_action: string
+}
+
+const VerifyAuth = () => {
+    const [success, setSuccess] = useState<Boolean>(false)
+    const [data, setData] = useState<AuthData | null>(null)
+    const [error, setError] = useState<Error | null>(null)
+
```

Comment:
Instead of having multiple states we can create complex states with [React useReducer Hook](https://reactjs.org/docs/hooks-reference.html#usereducer) or with [React Redux](https://react-redux.js.org/)